modules/tide_landing_page/js/hero_image_theme.js

```diff
@@ -0,0 +1,39 @@
+(function (Drupal) {
+  "use strict";
+
+  Drupal.behaviors.heroImageTheme = {
+    attach: function (context, settings) {
+      const imageTheme = document.querySelector(
+        'select[data-drupal-selector^="edit-field-landing-page-hero-theme"]'
+      );
+
+      if (imageTheme !== null) {
+        setImageTheme();
+
+        const headerStyles = document.querySelectorAll(
+          'input[data-drupal-selector^="edit-header-style-options"]'
+        );
+
+        if (headerStyles !== null) {
```

Comment:
The check if (headerStyles !== null) should instead check for headerStyles.length > 0. It’s better to check if any elements are found before adding listeners.